tests/rptest/services/utils.py

```diff
@@ -12,6 +12,10 @@ class BadLogLines(Exception):
     def __init__(self, node_to_lines):
         self.node_to_lines = node_to_lines
 
+    @staticmethod
+    def _get_hostname(node):
+        return node.account.hostname
```

Comment:
when `node` is a `dict` (in the `pod` case) I suspect this won't work because dict doesn't have `.` dot access?

Reply:
Yes, this is not the final version. I still think that initial approach was better. It would made updates to BagLog logic and Cloud Broker logic independently. But I agree that simpler is better than complex.

Reply:
Don't take my reviews a requests for things to be a certain way, they are at best suggestions, and may not even make sense :)

Reply:
Future me thinks that my current code is bad and should be refactored without remorse.